db/projects/kin/project.yaml

```diff
@@ -0,0 +1,53 @@
+name: Kin
+start_date: '2017-05-25'
+web:
+  - 'https://kin.kik.com'
+  - 'https://kinecosystem.org/'
```

Comment:
Please move primary web to first place - primary is kinecosystem.org

Reply:
OK, in this case both url goes to 'https://kinecosystem.org/'